### PR TITLE
Use css-property-sort-order-smacss

### PR DESCRIPTION
**Pull request by xxhomey19**

### CODE REVIEW

**Comment by kpman** [comment]
looks great

**Comment by kpman** [comment]
maybe after merge this, we should bump version to 1.0?

**Comment by jigsawye** [comment]
@kpman Agree.

File: order.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const smacssOrder = require('css-property-sort-order-smacss');
+
+const order = {};
+
+Object.keys(smacssOrder).forEach(category => {
+  let properties = [];
+
+  for (let i = 0; i < smacssOrder[category].length; i++) {
+    properties = properties.concat(smacssOrder[category][i]);
+  }
```
Comment (jigsawye) [inline order.js]:
```js
const properties = smacssOrder[category].reduce(

```

**Pull request merged**